packages/example-app-base/src/components/PlatformBar/PlatformBar.base.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { classNamesFunction, FocusZone, DefaultButton, Icon, TooltipHost } from 'office-ui-fabric-react';
+import { IPlatformBarProps, IPlatformBarStyleProps, IPlatformBarStyles } from './PlatformBar.types';
+import { IPlatform } from '../PlatformPicker/index';
+
+const getClassNames = classNamesFunction<IPlatformBarStyleProps, IPlatformBarStyles>();
+
+export class PlatformBarBase extends React.Component<IPlatformBarProps> {
```

Comment:
Since this doesn't have state:
```suggestion
export class PlatformBarBase extends React.PureComponent<IPlatformBarProps> {
```